main.js

```diff
@@ -23,13 +23,19 @@ function addGarment(event) {
   dressBear(event);
 }
 
-// Refactored functions from above
+// Refactored functions from below
 function toggleGarments(category, index) {
   if (event.target.classList.contains(category)) {
     toggleActiveClass();
-    toggleGarment(index);
+    placeGarment(index);
   }
 }
+// function toggleGarments(category, index) {
+//   if (event.target.classList.contains(category)) {
+//     toggleActiveClass();
+//     placeGarment(index);
+//   }
+// }
```

Comment:
This was the function I renamed. It was too close in name to the toggleGarments function that it sat in.